program_info/modrinth-mrpack-mime.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This isn't being used anywhere?

Reply:
This is for packagers to register this on install

Reply:
I am pretty sure that this should be installed by CMake then.